### ISSUE TITLE
Document how to invoke pytest

### ISSUE THREAD

**Issue by EwoutH** [averbraeck/pydsol-core#1]
This repository has a unusual structure with the extra layer directories of pydsol/core for both the src and tests folder. This makes running pytest non-trivial.

The docs state that can be pytest used:
https://github.com/averbraeck/pydsol-core/blob/972b4ee01d4cce91e55e8c491709a66bb0bd0111/src/pydsol/core/__init__.py#L48

But with all "standard" invokements of Pytest I get either an import error or a warning that no tests are found.

```
 ModuleNotFoundError: No module named 'pydsol'
```

With this directory structure, which command needs to be ran from which folder for pytest to work?

**Comment by averbraeck** [comment]
Good point. I copied the structure from several other Python projects, but it can probably be streamlined better I will see how I can improve it! Tips are always welcome.

Second point: The readthedocs pages still lack a good description how to install the project locally on your computer such that the module tests, coverage, demos, and code run best.

**Comment by EwoutH** [comment]
I think the big question is if this repository will include other components than the core.

If not, an easy fix would be to flatten the directory to `src/pydsol_core` and `test/pydsol_core`.

Best practice is to even don't include a `src` folder for the functional code. That would result in having `pydsol_core` and `test/pydsol_core`.

You could even consider dropping de `_core` suffix completely, and just call it `pydsol`. pydsol is still [available](https://pypi.org/project/pydsol/) on PyPI btw.

> Your library does not belong in an ambiguous src or python subdirectory.

**Comment by EwoutH** [comment]
I created a PR with a possible refactor and CI implementation which follows best practices and works easily with CI: #2
